livechat/customer/web/api/v33.py

```diff
@@ -0,0 +1,947 @@
+            payload = prepare_payload(locals())
+        return self.session.post(
+            f'{self.api_url}/mark_events_as_seen{self.query_string}',
+            json=payload,
+            headers=headers)
+
+    def accept_greeting(self,
+                        greeting_id: int = None,
+                        unique_id: str = None,
+                        payload: dict = None,
+                        headers: dict = None) -> httpx.Response:
+        ''' Marks an incoming greeting as seen.
+
+            Args:
+                greeting_id (int): ID of the greeting configured within the license to accept.
+                unique_id (str): ID of the greeting to accept. You can get it from the `incoming_greeting` push.
+                payload (dict): Custom payload to be used as request's data.
+                                It overrides all other parameters provided for the method.
+                headers (dict): Custom headers to be used with session headers.
+                                They will be merged with session-level values that are set,
+                                however, these method-level parameters will not be persisted across requests.
+
+            Returns:
+                httpx.Response: The Response object from `httpx` library,
+                                which contains a server’s response to an HTTP request. '''
+        if payload is None:
+            payload = prepare_payload(locals())
+        return self.session.post(
+            f'{self.api_url}/accept_greeting{self.query_string}',
+            json=payload,
+            headers=headers)
+
+    def cancel_greeting(self,
+                        unique_id: str = None,
+                        payload: dict = None,
+                        headers: dict = None) -> httpx.Response:
+        ''' Cancels a greeting (an invitation to the chat).
+            For example, Customers could cancel greetings by minimalizing the chat widget with a greeting.
+
+            Args:
+                unique_id (str): ID of the greeting to cancel. You can get it from the `incoming_greeting` push.
+                payload (dict): Custom payload to be used as request's data.
+                                It overrides all other parameters provided for the method.
+                headers (dict): Custom headers to be used with session headers.
+                                They will be merged with session-level values that are set,
+                                however, these method-level parameters will not be persisted across requests.
+
+            Returns:
+                httpx.Response: The Response object from `httpx` library,
+                                which contains a server’s response to an HTTP request. '''
+        if payload is None:
+            payload = prepare_payload(locals())
+        return self.session.post(
+            f'{self.api_url}/cancel_greeting{self.query_string}',
+            json=payload,
+            headers=headers)
```

Comment:
I guess we are missing `request_email_verification` method in all versions?
https://developers.livechat.com/docs/messaging/customer-chat-api/v3.3#request-email-verification